objects/src/accounts/account_id.rs

```diff
@@ -17,7 +17,7 @@ use core::{fmt, ops::Deref};
 /// Additionally, account IDs have the following properties
 /// - For fungible asset faucets account IDs are guaranteed to start with ONE.
 /// - For regular accounts, the last 3 bytes of the ID are guaranteed to be all ZEROs.
-#[derive(Debug, Clone, Eq, PartialEq)]
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
```

Comment:
Should we implement `Default` to facilitate id initialization?

Reply:
I don't think we should initialize IDs with all zeros. Ideally, we should create IDs only from known seeds such that when we hash the seed, the result meets certain properties. An ID consisting of all zeros likely has an unknown seed.